### PR TITLE
Docs hackathon tasks 49-52

### PR DESCRIPTION
**Pull request by blairwinans**

### CODE REVIEW

File: docs/manual/file-manager/control-downloads.en.md [hunk 1]
```diff
@@ -1,57 +1,63 @@
...
 
 ## Protect directory {#protect directory}
 
-When you create a new folder in Contao, it is accessible via HTTP by default including all subfolders. If you want to protect a folder, make sure that "Public" is not selected when you create the folder. If a directory is public, the folders and files in it cannot be protected.
+When you create a new folder in Contao's File Manager, it is accessible via HTTP by default including all subfolders.
+If you want to protect a folder, make sure that "Public" is not selected when you create the folder. If a directory is 
+made public, all the folders and files within it cannot be protected.
 
-![Protect directory](/de/file-manager/images/de/verzeichnis-schuetzen.png?classes=shadow)
+![Project directory](/de/file-manager/images/en/project_directory?classes=shadow)
```
Comment (Toflar) [inline docs/manual/file-manager/control-downloads.en.md]: Any reason you renamed "protect" to "project" here?

Comment (netzarbeiter) [reply docs/manual/file-manager/control-downloads.en.md]: don't forget .png

**Review by fritzmg — APPROVED** [review]

File: docs/manual/file-manager/control-downloads.en.md (hunk 1 above)
Comment (blairwinans) [reply docs/manual/file-manager/control-downloads.en.md]: Not enough coffee... will fix!

**Review by netzarbeiter — APPROVED** [review]
Thank you, Blair

**Pull request merged**